{"timestamp":"2025-09-12T06:11:33.453458","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-12T06:11:33.454314","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/indicium.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-12T06:11:33.926673Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T06:11:33.926900Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T06:11:33.927047Z","level":"info","event":"Current task name:extrair_csv","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T06:11:33.927123Z","level":"info","event":"Dag name:indicium","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T06:11:33.975519","level":"info","event":"Starting docker container from image meltano-banvic:latest","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:33.982724","level":"warning","event":"Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:35.545409","level":"info","event":"Usage: meltano [OPTIONS] COMMAND [ARGS]...","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:35.545786","level":"info","event":"Try 'meltano --help' for help.","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:35.545937","level":"info","event":"","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:35.546057","level":"info","event":"Error: No such command 'meltano'.","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:35.546177","level":"info","event":"","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:35.546287","level":"info","event":"Did you mean one of these?","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:35.546402","level":"info","event":"    elt","logger":"airflow.task.operators.airflow.providers.docker.operators.docker.DockerOperator"}
{"timestamp":"2025-09-12T06:11:35.935112","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"DockerContainerFailedException","exc_value":"Docker container failed: {'StatusCode': 2}","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":500,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":374,"name":"_run_image"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":435,"name":"_run_image_with_mounts"}],"is_group":false,"exceptions":[]},{"exc_type":"APIError","exc_value":"400 Client Error for http+docker://localhost/v1.51/containers/create: Bad Request (\"invalid mount config for type \"bind\": bind source path does not exist: /tmp/airflowtmp5f_wr5uh\")","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":365,"name":"_run_image"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":392,"name":"_run_image_with_mounts"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/docker/api/container.py","lineno":440,"name":"create_container"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/docker/api/container.py","lineno":457,"name":"create_container_from_config"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py","lineno":281,"name":"_result"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py","lineno":277,"name":"_raise_for_status"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/docker/errors.py","lineno":39,"name":"create_api_error_from_http_exception"}],"is_group":false,"exceptions":[]},{"exc_type":"HTTPError","exc_value":"400 Client Error: Bad Request for url: http+docker://localhost/v1.51/containers/create","exc_notes":[],"syntax_error":null,"is_cause":true,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py","lineno":275,"name":"_raise_for_status"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/requests/models.py","lineno":1026,"name":"raise_for_status"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-12T06:11:35.988068Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T06:11:35.988885Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T06:11:35.989089Z","level":"info","event":"Task:<Task(DockerOperator): extrair_csv>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T06:11:35.989196Z","level":"info","event":"Failure caused by Docker container failed: {'StatusCode': 2}","chan":"stdout","logger":"task"}
